crates/bevy_ecs/src/schedule/system_descriptor.rs

```diff
@@ -134,9 +149,12 @@ pub trait ParallelSystemDescriptorCoercion<Params> {
     /// Specifies that the system should run after systems with the given label.
     fn after<Marker>(self, label: impl AsSystemLabel<Marker>) -> ParallelSystemDescriptor;
 
+    /// Ignore all system-order ambiguities between this system and any other system
+    fn ignore_all_ambiguities(self) -> ParallelSystemDescriptor;
+
     /// Specifies that the system is exempt from execution order ambiguity detection
-    /// with other systems in this set.
-    fn in_ambiguity_set(self, set: impl AmbiguitySetLabel) -> ParallelSystemDescriptor;
+    /// with other systems with the given label.
+    fn ambiguous_with<Marker>(self, label: impl AsSystemLabel<Marker>) -> ParallelSystemDescriptor;
```

Comment:
```suggestion
    /// Allows ambiguous execution order between the system and all other systems.
    fn ignore_all_ambiguities(self) -> ParallelSystemDescriptor;

    /// Allows ambiguous execution order between the system and the system(s) with the given label.
    fn ambiguous_with<Marker>(self, label: impl AsSystemLabel<Marker>) -> ParallelSystemDescriptor;
```

Personally, I'd prefer calling these, like, `allow_ambiguous_with_all` and `allow_ambiguous_with` (or maybe `ignore_conflicts_with*`), but idk.

Reply:
I prefer this: it's much more natural to read in code, and the meaning is pretty esoteric either way.